desktop/app.js

```diff
@@ -3,9 +3,14 @@ var Menu = require( 'menu' );
 var BrowserWindow = require( 'browser-window' );	// Module to create native browser window.
 var path = require('path')
 
-module.exports = function main( url ) {
+require( 'module' ).globalPaths.push( path.resolve( path.join( __dirname ) ) );
+
+module.exports = function main() {
+	var url = 'file://' + path.join( __dirname, '..', 'dist', 'index.html' );
```

Comment:
I hate to sound naggy, but we should be careful here using `path.join()` to form a URL. Maybe it's not a problem, but of course the Windows path separator is a `\` and not a `/` and I'm not sure if that will cause issues. Maybe it's fine since it's `file://` but generally I would recommend specifically using a URL path joiner.